pkg/controller/template/render.go

```diff
@@ -222,6 +223,12 @@ func getPaths(config *RenderConfig, platformString string) []string {
 		platformBasedPaths = append(platformBasedPaths, platformOnPrem)
 	}
 
+	// If this is a cloud platform with DNSType set to `ClusterHosted` with
+	// LB IPs provided, include path for their CoreDNS files
+	if cloudPlatformLoadBalancerIPState(*config) == availableLBIPState {
+		platformBasedPaths = append(platformBasedPaths, cloudPlatformAltDNS)
```

Comment:
Before this, was anything processing the templates in `cloud-platform-alt-dns` ? Was is being done in bootstrap only?

Reply:
It was being done on the Bootstrap node successfully and while debugging for the CoreDNS pod not starting on the master nodes, I found this issue.